include/exadg/utilities/hypercube_resolution_parameters.h

```diff
@@ -77,7 +77,7 @@ fill_resolutions_vector(
   int    refine_level = 0;
   double n_cells      = 1.0;
 
-  while(n_cells <= std::pow(2, dim) * n_cells_max)
+  while(n_cells <= std::pow(2.0, dim) * n_cells_max)
```

Comment:
why didn't you switch to `int` for this file?

Reply:
Can you have a look at 3442131d?